components/raftstore-v2/tests/integrations/cluster.rs

```diff
@@ -339,6 +340,28 @@ impl RunningState {
 
         let background = Worker::new("background");
         let pd_worker = LazyWorker::new("pd-worker");
+        // Spawn a task to update the disk status periodically.
+        {
+            let tablet_registry = registry.clone();
+            let data_dir = PathBuf::from(tablet_registry.tablet_root());
+            let snap_mgr = snap_mgr.clone();
+            background.spawn_interval_task(cfg.value().pd_heartbeat_tick_interval.0, move || {
+                let snap_size = snap_mgr.total_snap_size().unwrap();
+                let mut kv_size = 0;
+                tablet_registry.for_each_opened_tablet(|_, cached| {
+                    if let Some(tablet) = cached.latest() {
+                        kv_size += tablet.get_engine_used_size().unwrap_or(0);
+                    }
+                    true
+                });
+                let used_size = snap_size + kv_size;
+                let (capacity, available) = disk::get_disk_space_stats(&data_dir).unwrap();
```

Comment:
Do we need add error handling to disk::get_disk_space_stats instead of using unwrap to prevent TiKV from panicking?

Reply:
Since this code is intended for testing purposes and not for formal use, it's not necessary to supply the check.

Reply:
Get. It's "v2" :)